classes/robot/crawler.php

```diff
@@ -548,52 +548,42 @@ public function process_queue($verbose = false) {
         if ($config->uselogs == 1) {
             $recentcourses = $this->get_recentcourses();
         }
+        // Grab the first item from the queue.
```

Comment:
the semantics of this function process_queue() have changed. This uses to process a queue, now it is not clear what it i doing. So this needs a rename or some phpdoc updates

Reply:
Has been updated